Gemfile

```diff
@@ -13,8 +13,9 @@ group :development, :test do
   gem 'climate_control'
 end
 
-gem 'azure_mgmt_resources', '~>0.5.0'
-gem 'azure_mgmt_compute',   '~>0.5.0'
-gem 'azure_mgmt_network',   '~>0.5.0'
-gem 'azure_mgmt_storage',   '~>0.5.0'
-gem 'haikunator',           '~>1.1'
\ No newline at end of file
+gem 'azure_mgmt_resources', '~>0.6.0'
+gem 'azure_mgmt_compute',   '~>0.6.0'
+gem 'azure_mgmt_network',   '~>0.6.0'
+gem 'azure_mgmt_storage',   '~>0.6.0'
+gem 'haikunator',           '~>1.1'
+
```

Comment:
It has been a practice in cordova/TS projects to ensure that there is a new line at the end. Actually it is one of the best practices in coding. One sample discussion on this: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline